tlf/handle_test.go

```diff
@@ -241,6 +285,30 @@ func TestHandleAccessorsPublic(t *testing.T) {
 		})
 }
 
+func TestHandleAccessorsSingleTeam(t *testing.T) {
+	w := []keybase1.UserOrTeamID{
+		keybase1.MakeTestTeamID(4).AsUserOrTeam(),
+	}
+
+	h, err := MakeHandle(
+		w, nil, nil, nil, nil)
+	require.NoError(t, err)
+
+	require.Equal(t, SingleTeam, h.Type())
+
+	// All types of IsWriter/IsReader calls should panic for team TLFs.
```

Comment:
Not really related to this PR; just thinking out loud:

Is this the plan (that tlf/handle.go panics on team) for the future as well? Do we use separate code for handling team membership through service?

If so, should they still live in the `tlf` package? Maybe it can accept an interface which does the actual checks on the membership. WDYT?